core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackVolumeAdaptor.java

```diff
@@ -0,0 +1,50 @@
+package de.danoeh.antennapod.core.service.playback;
+
+import de.danoeh.antennapod.core.feed.FeedMedia;
+import de.danoeh.antennapod.core.feed.FeedPreferences;
+import de.danoeh.antennapod.core.feed.VolumeReductionSetting;
+import de.danoeh.antennapod.core.util.playback.Playable;
+
+class PlaybackVolumeAdaptor {
```

Comment:
This name always reminds me of the `Adapter` classes you use for `RecyclerViews`. What about `PlaybackVolumeUpdater`?

Reply:
OK
```
class PlaybackVolumeUpdater {

       public void updateVolumeIfNecessary(..)
```